app/src/main/java/com/github/wnder/ImageFromGalleryActivity.java

```diff
@@ -67,6 +68,11 @@ private void changeText(Intent openGalleryIntent){
         }
     }
 
+    //can be kept for further use (see how it is handle in TakePictureActivity)
+    private boolean sendImageToDB(){
+        return true;
+    }
```

Comment:
Just why

Reply:
Ok je cède à la pression.

Reply:
I was wondering why didn't called this function: "returnTrue". Don't take it personally